src/fidesops/models/privacy_request.py

```diff
@@ -96,6 +97,23 @@ def cache_identity(self, identity: PrivacyRequestIdentity) -> None:
                     value,
                 )
 
+    def cache_encryption(
+        self, encryption_key: Optional[str] = None, nonce: Optional[str] = None
+    ) -> None:
+        """Sets the encryption key/nonce in the Fidesops app cache if provided"""
+        if not (encryption_key and nonce):
```

Comment:
I think this is unclear on bad input. e.g, it's probably more appropriate to raise an exception  if only one of these two is provided, since that's invalid input, which leads me to think that it'd be better to check this at the call site at `create_privacy_request`,